### DISCUSSION TITLE
Regarding Martini 3 Elastic network visualization issue:

### DISCUSSION THREAD

**Discussion by Kankanab** [Martini-Force-Field-Initiative/MartiniDiscussions#25] [Proteins]
Dear Sir,

I have converted the all-atom protein into CG resolution using martinize2 tool and used the command line as: 
martinize2 -f readyforcg1.pdb
![LHCII-AA-CG-mapped-modified](https://github.com/user-attachments/assets/86a2a338-ac17-436b-a09d-5da3f80519e0)
 -o monomer1_cg.top -x monomer1_cg.pdb -dssp /usr/bin/dssp -p backbone -ff martini3001 -elastic 

martinize2 -f readyforcg1.pdb -o monomer1_cg.top -x monomer1_cg.pdb -dssp /usr/bin/dssp -p backbone -ff martini3001 -elastic 
Now, if I visualize my cg structure using cg_bonds-v5.tcl script, then it is not properly creating the network or superimposing on aa structure. I have used in tkconsole: source cg_bonds-v5.tcl, 
cg_bonds -gmx gmx -top monomer1_cg.top -topoltype "elastic" -net "elastic", but not giving me the network properly. I am attaching the image for my trimeric protein system. It would be really great, if you kindly give me some suggestions. 

Thanks.

**Comment by csbrasnett** [comment]
Hi, I've recently developed a small package to help with visualising Martini topologies called [MartiniGlass](https://github.com/Martini-Force-Field-Initiative/MartiniGlass) that may be able to help.

However if I've understood your problem correctly, you're looking at your atomistic structure in vmd together with the elastic network that's been generated for the martini molecule. I think the elastic network's been generated fine, looking at the helices the bonds are as I'd expect.
